app/views/tag/show.html.erb

```diff
@@ -53,10 +53,10 @@
 
             <%= render partial: 'tag/show/contributors' %>
 
-            <div class="row">
+            <div class="row" style="padding-bottom:1rem;">
```

Comment:
Setting padding doesn't centre it perfectly. Flexbox is the way to go :)

Reply:
@Tlazypanda any update on this

Reply:
Actually @VladimirMikulic the row class has already display set to flex and so on checking further I found setting align-items to center wont work since the p tag has margin-top set as 0 but bottom margin was set to 1 rem hence I modified the margin top to 1 rem. Also the padding bottom is added as the normal bottom spacing had to be set to 1 rem inorder to fix the spacing. Not sure if this is the right approach but don't exactly know how else to set the flexbox :sweat_smile:   Here is the screenshot of my findings from the dev tools   
![Screenshot from 2020-03-25 15-28-16](https://user-images.githubusercontent.com/33183263/77524441-5accf300-6ead-11ea-8442-0be20ca0312c.png)
![Screenshot from 2020-03-25 15-27-53](https://user-images.githubusercontent.com/33183263/77524444-5c96b680-6ead-11ea-878e-1b6d89daaea5.png)

Reply:
@Tlazypanda `align-items: baseline` centres it perfectly :)

Reply:
@VladimirMikulic Thanks a lot! Have made the changes :smile:

Reply:
Perfect!